src/index.html

```diff
@@ -1,13 +1,50 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="container">
+    <div class="product-cards" data-qa="card">
+      <div class="product-cards__items">
+        <div class="image product-cards__image">
+          <img
+          src="images/imac.jpeg"
+          alt="Apple A1419"
+          class="image__item"
+        >
+        </div>
+        <h1 class="product-cards__text">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h1>
+        <p class="product-cards__code">Код товара: 195434</p>
+        <div class="stars product-cards__rate">
+          <span class="stars__image"></span>
+          <span class="stars__image"></span>
+          <span class="stars__image"></span>
+          <span class="stars__image"></span>
+          <span class="stars__image stars__image--disactive"></span>
+          <div class="stars_text">Отзывов: 5</div>
+        </div>
+        <div class="price product-cards__price">
+          <div class="price__text">Цена:</div>
+          <div class="price__value">69 999 грн</div>
+        </div>
+        <div class="button product-cards__button">
+          <a
+            href="#"
+            class="button__link"
+            data-qa="hover"
+          >
+            КУПИТЬ
+          </a>
+        </div>
```

Comment:
Don't create separate block for button. It's not independent functional block, it's just UI element.